packages/v1/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@esphome-webserver/v1",
+  "scripts": {
+    "build": "mkdir -p ../../_static/v1 && cp ./src/* ../../_static/v1/"
+  }
+}
```

Comment:
This conceptually replaces the copying that was previously happening via vite build process.